config/config.go

```diff
@@ -475,6 +476,11 @@ func (cfg *MetadataRegistryConfig) Validate() error {
 	return nil
 }
 
+// ValidatorBalancesConfig is the configuration for the validator balances analyzer.
+type ValidatorBalancesConfig struct {
```

Comment:
Why don't we use `ItemBasedAnalyzerConfig` directly for configs that only use `ItemBasedAnalyzerConfig` and no other fields? For example, we use `BlockBasedAnalyzerConfig` directly for block-based analyzers.

I think we would only want to do this if we wish to have custom validation for the config (which wouldn't match the default validation of the `ItemBasedAnalyzerConfig`), but that doesn't seem to be the case.

Reply:
We could definitely use the `ItemBasedAnalyzerConfig` directly; I think it's just become a habit out of laziness/convenience when copying over the analyzer boilerplate. In this case, I ended up adding a `start_height` parameter to the `ValidatorBalancesConfig` so I've kept the custom config struct. We can definitely update the other analyzers to use the item based config directly though.